src/applications/personalization/profile/tests/e2e/direct-deposit/v1-update-flow.cypress.spec.js

```diff
@@ -0,0 +1,98 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
```

Comment:
This entire file is mostly a copy-paste job from the existing `update-flow.cypress` test. It's just running against the old/existing version of Direct Deposit that only supports Direct Deposit for Comp and Pen.

Reply:
I had to add this test to make sure I wasn't breaking things that were in production since I was making changes that turned out to be non-trivial and a little bit scary :-D